engine/samples/settings/main.cpp

```diff
@@ -14,6 +14,15 @@ int main(int argc, char** argv)
 
     cubos.startupSystem("print setting value").after(settingsTag).call([](Settings& settings) {
         CUBOS_INFO("{}", settings.getString("greeting", "Hello!"));
+        settings.save("/settings.json", 4);
```

Comment:
```suggestion
        settings.save();
```
Maybe omit the arguments here? :thinking: or do you want to showcase them?